scripts/railgun-integration-tests.sh

```diff
@@ -1,6 +1,14 @@
 #!/bin/bash
```

Comment:
I don't understand the added value of this file. What makes
```console
$ ./scripts/railgun-integration-tests.sh
$ ./scripts/railgun-integration-tests.sh LEGACY
```
better than
```console
$ make test.integration
$ make test.integration.legacy
```
?

Also, this script currently won't work if called from any other working directory than the repository root. `make` doesn't have this problem because it has well-defined working directory semantics.

Recommandation: drop this script

Reply:
Following recommendation :+1: 2ebdd48216b9c6e0b7cb0d0fc3f027d86ed5b845